app.py

```diff
@@ -0,0 +1,139 @@
+from flask import Flask, render_template, redirect, Markup, url_for, jsonify, request
+from flask_cors import CORS, cross_origin
+import json
+import re
+from datetime import datetime as dt
+from pprint import pprint
+from package.broom import Player, Deck, Game, make_deck
+
+
+app = Flask(__name__)
+CORS(app, resources={r"/makedeck": {"origins": "http://localhost:8080"}})
```

Comment:
by adding something like the below, we can preempt endpoint constructions?

hard-paths are discouraged, and if we wrap them, then we can minimize their impact

```suggestion
def make_url(args, base="http://localhost:8080"):
    return base
    
CORS(app, resources={r"/makedeck": {"origins": make_url}})
```